packages/antd/src/widgets/AltDateWidget/index.tsx

```diff
@@ -14,11 +14,18 @@ export default function AltDateWidget<
   T = any,
   S extends StrictRJSFSchema = RJSFSchema,
   F extends FormContextType = any,
->(props: WidgetProps<T, S, F>) {
-  const { autofocus, disabled, id, name, onBlur, onFocus, options, readonly, registry } = props;
+>({
+  autofocus = false,
+  disabled = false,
+  options = { yearsRange: [1900, new Date().getFullYear() + 2] },
```

Comment:
Defaults can behave differently around object props than destructure initialization, due to the code that is in the `getWidget()` implementation in `@rjsf/utils`. You may want to deal with that too... See `mergeWidgetOptions()` in `getWidget.ts`